cypress/integration/application/index.js

```diff
@@ -39,3 +40,31 @@ describe('Application', () => {
     );
   });
 });
+
+describe('Application unknown route error pages', () => {
+  if (Cypress.env('APP_ENV') === 'local') {
+    var unknownRoutes = [
+      '/foobar',
+      '/foobar.amp',
+      '/igbo/foobar',
+      'igbo/foobar.amp',
+    ];
+    unknownRoutes.forEach(function(url) {
+      it('should display a news canonical error page', () => {
+        cy.visit(url, { failOnStatusCode: false });
+        let platform = 'news'
```

Comment:
```
const service = url.includes('igbo') ? 'igbo' : 'news'
```

Bit cleaner imo, and `service` is more consistent across simorgh than `platform`

Reply:
yeah looks cleaner and less lines of code

Reply:
Why do we want to assert for 400?

Reply:
Incase we somehow start serving our 404 error page via a 500 status code. (its a 500, but it says 404 in the html)
Adding a `cy.testResponseCode` above the `visit` should do it :)

Reply:
Done